text/0016-named-tracers.md

```diff
@@ -66,22 +76,46 @@ Examples (based on existing contribution libraries from OpenTracing and OpenCens
 
 ## Internal details
 
-By providing a `TracerFactory` / `MeterFactory` and *Named Tracers / Meters*, a vendor or OpenTelemetry implementation gains more flexibility in providing Tracers and Meters and which attributes they set in the resulting Spans and Metrics that are produced.
+By providing a `TracerProvider` / `MeterProvider` and *Named Tracers / Meters*, a vendor or OpenTelemetry implementation gains more flexibility in providing Tracers and Meters and which attributes they set in the resulting Spans and Metrics that are produced.
 
 On an SDK level, the SpanData class and its Metrics counterpart are extended with a `getLibraryResource` function that returns the resource associated with the Tracer / Meter that created it.
 
+## Glossary of Terms
+
+#### Reporting library
+Also known as the instrumentation library or the trace/metrics reporter, this may be either a library/module/plugin provided by OpenTelemetry that instruments an existing library, a third party integration which instruments some library, or a library that has implemented the OpenTelemetry API in order to instrument itself. In any case, the reporting library is the library which provides tracing and metrics data to OpenTelemetry.
+
+examples:
+* `@opentelemetry/plugin-http`
+* `io.opentelemetry.redis`
+* `redis-client` (in this case, `redis-client` has instrumented itself with the OpenTelemetry API)
+
+#### Tracer name / Meter name
+When it is acquired from the tracer/meter Provider, the tracer/meter is assigned a name and a version which is the name and version of the reporting library that acquired the tracer. In cases where a library has instrumented itself using the OpenTelemetry API, they may be the same.
```

Comment:
This first sentence seems to highlight the some of the confusion I was anticipating this PR would address. Saying that a tracer/meter is assigned a "name and version" and then referring to that assigned value as a "name" is semantically confusing.

Is a tracer/meter name a 2-tuple of a library name and version, or just the library name? The second sentence, and, to a greater extent, the following example seem to treat it as only the name.

I think [the point made by @jmacd](https://github.com/open-telemetry/oteps/pull/76#issuecomment-572252679) is going to be something needed to clarify named tracers and meters and an important thing to address in this PR. The semantic importance of calling their identifiers "names" is leading to a fair amount of confusion. Using a more generalized term like "identifier" (or "id") seems important to avoid semantic confusion, but also to help remove undesired associations (namespaces).

Reply:
What is currently referred to as a tracer/meter name is intended to be the 2-tuple of library name _and_ version. I can see the confusion and also why this might be mixed up as a namespace, which is why I agree that rephrasing it from _tracer/meter name_ to something better would make sense.

Reply:
@MrAlias can you take a look at https://github.com/open-telemetry/oteps/pull/76/commits/312a65f055ce48cf6b21b8084c0ea9979bda3a69 and see if that helps?

Reply:
@dyladan the changes look good.

One minor change I would suggest is to the section header:

 ```diff
- #### Tracer name / Meter name and version
+ #### Tracer / Meter name and version
```

NBD, if you don't think it worth changing (given this is only an OTEP and not the spec).